src/Compilers/Test/Core/MarkedSource/MarkupTestFile.cs

```diff
@@ -50,7 +50,7 @@ public static class MarkupTestFile
             RegexOptions.Multiline | RegexOptions.IgnorePatternWhitespace);
 
         private static void Parse(
-            string input, out string output, out int? position, out IDictionary<string, ArrayBuilder<TextSpan>> spans)
+            string input, bool treatPositionIndicatorsAsCode, out string output, out int? position, out IDictionary<string, ArrayBuilder<TextSpan>> spans)
```

Comment:
needed to update our internal test harness to allow the test to configure how `$$` is treated.  For raw-string tests, we do not want to treat it as the caret.